meta/documents/changelog_en.md

```diff
@@ -7,6 +7,8 @@
 - It's no longer possible to submit a plentyShop contact form without accepting the privacy policy.
 - On mobile, display errors no longer inhibit the usability of the permanently displayed search.
 - On mobile, you can now close the language detection from the **Language detection** widget again.
+- On mobile, the images in the **Article grid** widget are displayed correctly again with the setting: "Number of articles per row (mobile): 2".
```

Comment:
```suggestion
- On mobile, the images in the **Article grid** widget are displayed correctly again with the setting: **Number of articles per row (mobile): 2**.
```
**suggestion (non-blocking): Formatting**

All UI elements should be in bold. Doesn't matter if it's the name of a widget, a label or a setting.

Reply:
```suggestion
- On mobile, the **Article grid** widget displays images correctly again with the setting: **Number of articles per row (mobile): 2**.
```
**nitpick: Active voice**

Prefer active voice in English.